mysql-async/src/main/java/com/github/jasync/sql/db/mysql/binary/encoder/DateTimeEncoder.kt

```diff
@@ -2,14 +2,13 @@ package com.github.jasync.sql.db.mysql.binary.encoder
 
 import com.github.jasync.sql.db.mysql.column.ColumnTypes
 import io.netty.buffer.ByteBuf
-import org.joda.time.LocalDateTime
-import org.joda.time.ReadableDateTime
+import java.time.OffsetDateTime
 
 object DateTimeEncoder : BinaryEncoder {
     override fun encode(value: Any, buffer: ByteBuf) {
-        val instant = value as ReadableDateTime
+        val zdt = value as OffsetDateTime
```

Comment:
isn't it more suitable to use ZonedDateTime here?

Reply:
Apparently, it is recommended to be used when storing to database. 
doc: https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html
A good explanation: https://stackoverflow.com/a/30234992/5375462

Reply:
I was mostly referring [this](https://blog.joda.org/2014/11/converting-from-joda-time-to-javatime.html) that said this is the equivalent for joda DateTime.

Reply:
the doc you referenced states both are acceptable, no?

Reply:
yes, only that `ZonedDateTime` is semantically the same as `DateTime` while the other is similar.

Reply:
well I could change it to Zoned but even official java doc says to use Offset for saving into database and thats what https://github.com/pgjdbc/r2dbc-postgresql/ does for reference.

Reply:
ok, let's keep it with OffsetDateTime